jsonschema-core.xml

```diff
@@ -2350,9 +2350,9 @@
                         <t>
                             Validation succeeds if each element of the instance validates
                             against the schema at the same position, if any.  This keyword
```

Comment:
While we're here, should we update this to say "subschema" instead of "schema?"

Reply:
good catch, done!